zlibWrapper/Makefile

```diff
@@ -7,8 +7,9 @@
 
 # Paths to static and dynamic zlib and zstd libraries
 ifneq (,$(filter Windows%,$(OS)))
-STATICLIB = ../../zlib/libz.a ../lib/libzstd.a
-IMPLIB    = ../../zlib/libz.dll.a ../lib/libzstd.a
+ZLIBDIR = ../../zlib1.2.8
```

Comment:
It must be set manually. I need to check if MSYS supports -lz

There is more issues with Windows. For example the last separate Windows binary is zlib 1.2.3:
http://gnuwin32.sourceforge.net/packages/zlib.htm